config/v1/types_project.go

```diff
@@ -19,9 +18,20 @@ type Project struct {
 	Status ProjectStatus `json:"status"`
 }
 
+// TemplateReference references a project request template in a 'openshift-config' namespace.
```

Comment:
Per @derekwaynecarr review of authn APIs, this should state that the point of use determines the namespace.